Dockerfile

```diff
@@ -7,11 +7,46 @@ RUN apk add --update-cache \
 WORKDIR /app
 
 COPY . .
+
 RUN mix local.hex --force
 RUN mix local.rebar --force
 RUN mix deps.get
 RUN npm --prefix assets install
 RUN mix compile
 
 RUN ["chmod", "+x", "./entrypoint.sh"]
-ENTRYPOINT ["sh", "./entrypoint.sh"]
\ No newline at end of file
+ENTRYPOINT ["sh", "./entrypoint.sh"]
+
+# https://hexdocs.pm/phoenix/releases.html
+FROM development AS build
```

Comment:
I would suggest not starting from the development image. I don't see any benefits because you are compiling and downloading everything again anyways. It will also take more time, because for the release image you have to go through the development setup first.

Reply:
Done.